dist/spec/payid-easy-checkout.txt

```diff
@@ -0,0 +1,728 @@
+     5.2.  Access Control  . . . . . . . . . . . . . . . . . . . . .   8
+   6.  IANA Considerations . . . . . . . . . . . . . . . . . . . . .   8
+   7.  Acknowledgments . . . . . . . . . . . . . . . . . . . . . . .   8
+   8.  Appendix  . . . . . . . . . . . . . . . . . . . . . . . . . .   8
+     8.1.  Motivation  . . . . . . . . . . . . . . . . . . . . . . .   8
+       8.1.1.  Design Goals  . . . . . . . . . . . . . . . . . . . .   9
+     8.2.  Example Usage . . . . . . . . . . . . . . . . . . . . . .   9
+       8.2.1.  PayID Easy Checkout Initiation  . . . . . . . . . . .   9
+       8.2.2.  PayID Easy Checkout Wallet Discovery  . . . . . . . .   9
+       8.2.3.  Assemble PayID Easy Checkout URL with Query
+               Parameters  . . . . . . . . . . . . . . . . . . . . .  10
+       8.2.4.  Redirect Payer to Their Wallet  . . . . . . . . . . .  11
+       8.2.5.  Payer Confirms Payment  . . . . . . . . . . . . . . .  11
+   9.  References  . . . . . . . . . . . . . . . . . . . . . . . . .  11
+     9.1.  Normative References  . . . . . . . . . . . . . . . . . .  11
+     9.2.  Informative References  . . . . . . . . . . . . . . . . .  12
+     9.3.  URIs  . . . . . . . . . . . . . . . . . . . . . . . . . .  12
+   Authors' Addresses  . . . . . . . . . . . . . . . . . . . . . . .  12
+
+1.  Terminology
+
+   This protocol can be referred to as the "PayId Easy Checkout
+   Protocol".  It uses the following terminology:
+
+   o  PayID client: a client that queries a PayID server using the PayID
+      Protocol as defined in [PAYID-PROTOCOL][].
+
+
+
+
+Kramer, et al.          Expires January 24, 2021                [Page 2]
+
+Internet-Draft   Draft 1 - PayID Easy Checkout Protocol        July 2020
+
+
+   o  PayID server: the endpoint that returns payment account(s)
+      information, which conforms to the PayID Protocol.
+
+   o  Recipient: Individual or entity receiving a payment (e.g.,
+      e-commerce merchant, charity).
+
+   o  Payer: Individual or entity originating a payment to a
+      "recipient".
+
+   o  Wallet: A device or application that holds funds (may be a non-
+      custodial wallet).
+
+   o  PayID Easy Checkout URL: The URL that is the result of the PayID
+      Easy Checkout protocol; can be used to redirect a client to a
+      wallet corresponding to a particular PayID.
+
+   The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
+   "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and
+   "OPTIONAL" in this document are to be interpreted as described in
+   [RFC2119] and [RFC9174][].
+
+2.  Introduction
+
+   The PayID Easy Checkout Protocol is a minimal protocol that allows a
+   recipient (e.g., an online merchant or a charity) to request a
+   payment from a payer using only the payer's PayID.  Implementations
+   of the protocol should require little to no server-side engineering
+   efforts, while creating an improved and uniform user experience for
+   payers.
```

Comment:
Improved and uniform user experience compared to what? It will be useful to add reference to some of the existing check-out flows and how this one adds to better user experience for payers.

Reply:
I'm changing the wording to "a seamless and uniform user experience for payers".  I want to limit the amount of "selling" in the RFC, and focus more on providing the bare minimum requirements to enable Easy Checkout, so comparing this protocol to existing flows seems unnecessary.